app/models/concerns/auction_scopes.rb

```diff
@@ -4,6 +4,10 @@ module AuctionScopes
   included do
     scope :accepted, -> { where(status: statuses['accepted']) }
     scope :accepted_pending_payment_url, -> { where(status: statuses['accepted_pending_payment_url']) }
+    scope :accepted_and_pending, lambda {
```

Comment:
According to our Rubocop, we use lambda for multiple-line lambdas (the `->` is just syntactic sugar), so that's why I added it